blog/2024-06-03-ratify-1.2.0.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: ratify-v1.2.0-is-available
+title: Ratify v1.2.0 is available! 
+authors: [ratify]
+tags: [verification]
+---
+
+We are thrilled to announce the release of Ratify v1.2.0! 
+
+Ratify is a verification framework that ensures all supply chain artifacts stored in a registry are thoroughly vetted and trustworthy. By [integrating Ratify into your Kubernetes](https://notaryproject.dev/blog/2023/announcing-major-release/#integration-with-admission-controller-for-kubernetes-usage) workflow, you can protect your deployment pipeline from potential threats and vulnerabilities, significantly enhancing the overall security of your applications.
+
+Read on to discover the exciting new features and how they can benefit you!
+
+## Key Features, Contributions, and Decprecations
+
+Key features in this release include:
+- Cosign Verifier Enhancements 
+- Kubernetes Multi-Tenancy Support
+- Key Management Provider
+
+Contributions from the community:
```

Comment:
This may confuse readers that there were only two contributors in this release but actually not. To show our gratitude to contributors, I would suggest moving it to [the last paragraph](https://github.com/ratify-project/ratify-web/pull/83/files#diff-49c0b5ac2af0556855621e030ebce29cace817941db93ca8eb947cac842671c2R62). 

A sample acknowledgement:

> The Ratify release team wants to thank the entire Ratify community for all the activity and engagement that has been vital for helping the project grow and reach this important milestone. We are glad to see two new contributors engaged in this release:

Reply:
Would it be best just to remove that section? Shorter is often always better when it comes to written content. Perhaps it's best to send a message in slack to celebrate community contributions?

Reply:
Removing this section sounds good to me. Let's get this PR merged into main first to publish it and improve it later if any.